### ISSUE TITLE
Update docs to use bin/logstash-plugin

### ISSUE THREAD

**Issue by markwalkom** [elastic/logstash#5080]
Running latest and went to install the translate filter; 

```$ logstash-2.3.1/bin/plugin list
The use of bin/plugin is deprecated and will be removed in a feature release. Please use bin/logstash-plugin```

[The docs](https://www.elastic.co/guide/en/logstash/current/working-with-plugins.html#working-with-plugins) still use `bin/plugin` so we just need to update that :)

**Comment by purbon** [comment]
Good point with that, thanks for your time man!

**Comment by purbon** [comment]
@dedemorton easy to update somehow, isn't?

**Comment by dedemorton** [comment]
@purbon Yah, it's an easy fix...already on my radar. The change was implemented in the alpha docs, but not 2.3. There's no easy way to cherry-pick the changes, though, because the commit includes code changes.
